src/ucs/type/spinlock.h

```diff
@@ -89,7 +89,7 @@ static inline void ucs_spin_unlock(ucs_spinlock_t *lock)
 {
     --lock->count;
     if (lock->count == 0) {
-        lock->owner = 0xfffffffful;
+        lock->owner = (pthread_t)0xfffffffful;
```

Comment:
the same, may be use PTHREAD_T_NULL instead of -1?

Reply:
Yep.